Dockerfile.clients.rh

```
@@ -0,0 +1,96 @@
+# Provides the Trusted Artifact Signer CLI binaries, cosign and gitsign
+FROM quay.io/securesign/cli-cosign@sha256:c0187868dbf5c7dcaa545a07a194ef90b50655df497db3f7f0de0a3c4eaa41f7 AS cosign
+FROM quay.io/securesign/gitsign@sha256:3229dc9d5bad1344663ac92a980eb6abcd5715dc3812a20a2129f60b885ececf AS gitsign
+
+# Provides the Trusted Artifact Signer CLI binary, fetch-tsa-certs
+FROM quay.io/securesign/fetch-tsa-certs@sha256:04ee10dd6f36b7ebca80c0e7badeb5c69d4ae2b37eb1abbea204d1af4eb1d0cc as fetch_tsa_certs
+
+# Provides the Trusted Artifact Signer CLI binaries, rekor-cli and ec
+FROM quay.io/securesign/rekor-cli@sha256:285603d7aba24ecd88d98afb20807968a12557fd33a31c52b57df528c3cf57c4 as rekor
+FROM quay.io/redhat-user-workloads/rhtap-contract-tenant/ec-v04/cli-v04@sha256:013fed3832c831cfa45ecad66ba335ebb0438ade168174d474c0ed1ac3c2c59c as ec
+
+# Provides the Trusted Artifact Signer CLI binaries trillian-createtree and trillian-updatetree
+FROM quay.io/securesign/trillian-createtree@sha256:2a17108678e51bf39d80b3a7fc577ec9c12de10e19286e3e5298fb8cfcf9309c as trillian-createtree
+FROM quay.io/securesign/trillian-updatetree@sha256:cae560a63bc4f6aae4c8d5110ca903f52518be7b7db34ddebf29a541f04c6c45 as trillian-updatetree
+
+FROM registry.access.redhat.com/ubi9/httpd-24@sha256:f6a99e33d5044e6214578a5824d069de4086f70e50e7b856c3cfce1819150ec9
+ENV APP_ROOT=/opt/app-root
+WORKDIR $APP_ROOT/src/
+
+RUN mkdir -p "/var/www/html/darwin" && \
+    mkdir -p "/var/www/html/linux" && \
+    mkdir -p "/var/www/html/windows"
+
+# Copy the cosign binaries from the previous stages
+COPY --from=cosign /usr/local/bin/cosign-darwin-amd64.gz  /var/www/html/darwin/cosign-amd64.gz
```

```suggestion
COPY --from=cosign /usr/local/bin/cosign-darwin-amd64.gz  /var/www/html/clients/darwin/cosign-amd64.gz
```

The image looks fine, one thing though, in the operator the cli images are on the path `/clients/<arch>/%s-amd64.gz` so we need to add it here to all the paths. https://github.com/securesign/secure-sign-operator/blob/main/internal/clidownload/component.go#L185

Ofc if you dont want to make the changes we can always change the path in the operator

---

No problem -- find and replace is easy ;)